src/main/java/thePackmaster/powers/maridebuffpack/RecurringThemePower.java

```diff
@@ -0,0 +1,53 @@
+package thePackmaster.powers.maridebuffpack;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.NonStackablePower;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import thePackmaster.cards.maridebuffpack.RecurringTheme;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.util.maridebuffpack.OnDebuffLossPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class RecurringThemePower extends AbstractPackmasterPower implements OnDebuffLossPower, NonStackablePower {
```

Comment:
If Recurring Theme was double-played, you would end up with 1 copy in the exhaust pile while the effect should add 2 to your hand, cases like this would mean the player needs something to properly track the information.

Reply:
👍🏻